spec/index.bs

```diff
@@ -13714,6 +13726,52 @@ In WebGPU, it does so by [$Replace the drawing buffer|replacing the drawing buff
     validation will fail inside {{GPUCanvasContext/getCurrentTexture()}}.
 </div>
 
+<h3 id=gpucanvascolormode data-dfn-type=enum>`GPUCanvasColorMode`
+<span id=GPUCanvasColorMode></span>
```

Comment:
Span not needed for new headings (only needed for old ones that changed id, to preserve historical hyperlinks).
```suggestion
```